### ISSUE TITLE
Cookies/settings ignored for DuckDuckGo site

### ISSUE THREAD

**Issue by rugk** [myano/lmddgtfy#14]
STR:
1. Change the settings (e.g. theme) in DuckDuckGo.
2. Use/Click (on) a lmddgtfy link.

What happens: The search result page is shown in the default theme/with default settings.
What should happen: Just show the site as usual.

Possible reason: Maybe DuckDuckGo uses [strict same site cookies](https://scotthelme.co.uk/tough-cookies/#samesitecookies) for storing the preferences and thus GET request from third-parties (= not "same origin") are not allowed to get access to cookies.

Firefox 53.0

**Comment by zystvan** [comment]
It doesn't seem possible to do this reliably to me, because, as I understand it, you would have to access LMDDGTFY directly from DDG by clicking a link on DDG's website. Typing it into your URL bar wouldn't work, because then you wouldn't be being referred by DDG. This would be an inconvenient way to transfer your theme, and could be done easier by just clicking the correct button on LMDDGTFY instead after navigating there however you choose. 

And that's assuming that DDG allows cookies to be transferred under at least some circumstances, which seems unlikely to me since DDG is privacy-focused and wouldn't want to share cookies with 3rd-parties. 

I don't think this is a bug, and I don't think it's reasonably possible to change this behavior regardless of how DDG handles its cookies.

**Comment by rugk** [comment]
Wait, wait, wait. First: Is my assumption about the cookies correct?
Secondly the bug is just that the user does not see the settings/theme. That's unexpected.

**Comment by zystvan** [comment]
Yes, it looks to me like DDG is using strict same-site cookies. 

> Secondly the bug is just that the user does not see the settings/theme. That's unexpected.

I was aware of this while I built the current version of LMDDGTFY, so not really unexpected. There's no possible way (that I'm aware of) to reliably sync settings between DDG and LMDDGTFY. Unless one of us can find a way to do this I think this issue can be closed. 

@myano Do you have any thoughts on this?
